### PR TITLE
Fix Windows build with submodules

### PR DESCRIPTION
**Pull request by gasinvein**
Follow-up for #415

### CODE REVIEW

File: .clang-format-ignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+src/third-party/*
+data/third-party/*
```
Comment (Shatur) [inline .clang-format-ignore:2]: Maybe remove this line? It doesn't contain any C++ code.

```suggestion
data/third-party/*
```

**Review by Shatur — CHANGES_REQUESTED** [review]
Thank you. Let's see if CI passes.

**Comment by Shatur** [comment]
Thanks!!!

**Comment by gasinvein** [comment]
Sorry for the bit of a mess. I probably would've noticed something is off with the submodules versions if the windows build job didn't repeatedly hang in my fork.

**Comment by Shatur** [comment]
It's fine. Windows is always a mess.

**Pull request merged**